flowcore/src/main/java/com/raizlabs/android/dbflow/sql/QueryBuilder.java

```diff
@@ -164,6 +167,34 @@ public QueryClass appendNotEmpty(String string) {
         return castThis();
     }
 
+    /**
+     * Appends ON DELETE or ON UPDATE Action defined in ForeignKeyReference.
+     *
+     * @param action defined {@link com.raizlabs.android.dbflow.annotation.Action}
+     * @return QueryBuilder
+     */
+    public QueryBuilder appendAction(Action action) {
```

Comment:
I would do something like ```action.name().replace("-"," ")``` instead to keep it simple and it does pretty much the same as here. Also I would add it as a method in the ```TableCreationQueryBuilder``` instead and use it vs. a ```QueryBuilder``` in this scenario.